### PR TITLE
Added option to ignore missing files (don't show warining) when loadi…

### LINKED ISSUE #113: Option to ignore non-found keys during autoloading
I use the same KeePass db on multiple devices and thus have different keys on each of them.
However I'd still like for KeeAgent to automatically load these keys, currently this kind of works but one has to click away all the errors every time the db is loaded.

I thought about this for a bit and came up with two solutions: Either a per-device setting for key loading, which would be a lot harder to set up since KeePass (afaik) doesn't have any per-device settings, or alternatively **an option to simply ignore "File not found" errors when loading keys on KeePass startup.**

I think it would make more sense to do this only for that one kind of error so users still get notified when something else goes wrong.

This could either be done per-entry or globally, my opinion is that globally would make more sense as someone who needs this would probably enable it for most keys anyways.

### PR DESCRIPTION
**Pull request by KuttKatrea**
Added global option to ignore missing files (don't show warnings).
Resolves #113

### CODE REVIEW

File: KeeAgent/KeeAgentExt.cs [hunk 1]
```diff
@@ -80,6 +80,7 @@ public sealed partial class KeeAgentExt : Plugin
     const string unixSocketPathOptionName = pluginNamespace + ".UnixSocketPath";
     const string userPicksKeyOnRequestIdentitiesOptionName =
       pluginNamespace + ".UserPicksKeyOnRequestIdentities";
+    const string ignoreMissingFilesName = pluginNamespace + ".IgnoreMissingFilesName";
```
Comment (dlech) [inline KeeAgent/KeeAgentExt.cs]: Can we call this `.IgnoreMissingExternalKeyFiles`?

Comment (KuttKatrea) [inline KeeAgent/KeeAgentExt.cs]: Right, I'll make the change and update the pull request

**Comment by KuttKatrea** [comment]
Changed `IgnoreMissingFiles` for `IgnoreMissingExternalKeyFiles` and rebased onto current `master`.

**Comment by dlech** [comment]
Thanks! I added a new options group since this option applies to more than just Agent mode and squashed the commits. b0eed2687ee74add4d10bfc6988bdd890487888e

**Pull request closed without merging**